apps/dh/api-dh/source/DataHub.WebApi/Modules/ProcessManager/Requests/Client/IRequestsClient.cs

```diff
@@ -12,13 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using Energinet.DataHub.WebApi.GraphQL.Enums;
+using Energinet.DataHub.ProcessManager.Orchestrations.Abstractions.Processes.Shared.BRS_026_028;
 
-namespace Energinet.DataHub.WebApi.GraphQL.Types.Calculation;
+namespace Energinet.DataHub.WebApi.Modules.ProcessManager.Requests.Client;
```

Comment:
Suggestion: It might be be better to delete, and create a new file instead of renaming, when its all new logic. It is abit confusing when reviewing :-)

Reply:
I agree, it is super confusing, but I think it's just git/GitHub being weird sometimes? AFAIK git does not track renames, but uses an algorithm to detect similarities. So renaming or delete -> create is all the same to git.